### PR TITLE
Shown Error messages using Snackbars

### PR DESCRIPTION
**Pull request by codeDamon**
Replaced error Toasts with Snackbars.

# Related Issue
Issue #8 Solved

# Proposed Changes
- Replaced Toasts showing error message with Snackbars   

# Screenshots

Updated (Example)
screenshot

### CODE REVIEW

File: app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java [hunk 1]
```diff
@@ -109,6 +112,7 @@ public void onComplete(@NonNull Task<AuthResult> task) {
                             int selectedId = userselected.getCheckedRadioButtonId();
                             selection = (RadioButton) findViewById(selectedId);
                             Toast.makeText(getApplicationContext(),selection.getText().toString(),Toast.LENGTH_SHORT).show();
```
Comment (plazzy99) [inline app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java:114]: comment this

**Review by plazzy99 — APPROVED** [review]
👍🏻 
Do this change

File: app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java (hunk 1 above)
Comment (codeDamon) [reply app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java:114]: Should I comment all three lines i.e. 112,113,114 or just Toast line(114)?

Comment (plazzy99) [reply app/src/main/java/com/CodeNaroNa/vendor/relief/Activity/SignUp.java:114]: no we are using selection later in the code i guess,

**Comment by codeDamon** [comment]
Done Changes!

**Review by plazzy99 — APPROVED** [review]

**Pull request merged**